internal/build/build.go

```diff
@@ -26,6 +28,57 @@ type Builder interface {
 	GID() int
 	LifecycleDescriptor() builder.LifecycleDescriptor
 	Stack() builder.StackMetadata
+	Image() imgutil.Image
+}
+
+type mountPaths struct {
```

Comment:
nit: Could we move this to the bottom of the file, below the exported funcs/interfaces?

Reply:
Sure